sentences/en/climate_HassClimateGetTemperature.yaml

```diff
@@ -2,18 +2,19 @@ language: "en"
 intents:
   HassClimateGetTemperature:
     data:
-      # What is...
+      # Get temperature of a climate in the same area as a satellite device
+      - sentences:
+          - "<what_is> [the] [current] <temp> [[in] here]"
+          - "how (hot|cold|warm|cool) is it [[in] here]"
+        requires_context:
+          area:
+            slot: true
+
+      # Get temperature of a climate in an area or with a name
       - sentences:
           - "<what_is> [the] [current] <temp> [of] <name>"
-          - "<what_is> [the] [current] <temp> [in <area>]"
+          - "<what_is> [the] [current] <temp> in <area>"
           - "<what_is> <area> [current] <temp>"
           - "<what_is> <name> [current] <temp>"
-      # How...
-      - sentences:
-          - "how (hot|cold|warm|cool) is it [in <area>]"
+          - "how (hot|cold|warm|cool) is it in <area>"
           - "how (hot|cold|warm|cool) is <name>"
```

Comment:
This same sentence could work for an area or floor too. Maybe `(<name>|<area>|<floor>)` would do.

Reply:
Actually, this is pretty close to the sentence above. Maybe they can be combined to be `how (hot|cold|warm|cool) is (<name>|[it] <in_area_floor>)`.

Reply:
Combined: [9177e6..74a7e7](https://github.com/home-assistant/intents/compare/9177e6b793e0cd84f02c16fe43986b47ac04bd42..00c49802397480507cf5d5d65f81d1ce14ffcebf)

I've made `[it in]` optional to allow "how hot is the living room?". Not sure if this would introduce some ambiguity with `<name>` though.